packages/app/src/App.tsx

```diff
@@ -66,6 +67,7 @@ const AppRoutes = () => (
       path="/tech-radar"
       element={<TechRadarRouter width={1500} height={800} />}
     />
+    <Route path="/graphiql" element={<GraphiQLRouter />} />
```

Comment:
GraphiQL isn't a part of the create-app provided plugins?

Reply:
Nop, it's not